src/Suave/Http.fs

```diff
@@ -415,7 +415,11 @@ module Http =
       socketBinding : SocketBinding }
 
     member x.uri (path : string) query =
-      let path' = if path.StartsWith "/" then path else "/" + path
+      let path' =
+        match Uri.TryCreate(path, UriKind.Absolute) with
+        | true, uri -> uri.AbsolutePath |> Uri.UnescapeDataString
```

Comment:
It looks great, thanks. If you could just avoid using the pipe here we can merge it.